clj-kondo/clj-kondo.exports/clara/rules/hooks/clara_rules.clj

```diff
@@ -0,0 +1,393 @@
+(ns hooks.clara-rules
```

Comment:
Wow. I’m surprised to see these detailed hooks written instead of just ignoring the forms. Nice.

Reply:
Yeah the main value being added with this is not ignoring the forms but actually transforming them to forms that the linter can understand and give you meaningful unresolved-symbol/unresolved-var etc warnings/errors.

Reply:
Yeah I didn’t realize the full scope of this PR until your explanations. Nice. 

I think this makes sense.

Reply:
Agreed, this looks pretty extensive and I'll be curious to take a closer look (I've been away for a bit and not looking at this) but it looks good.  To be clear, I don't want to block on my looking at it.

Reply:
Sounds good, I'm happy to make some further improvements later then if you find any issues, but based on dog-fooding this on my own library of rules it is working very nicely.